slasher/detection/detect_test.go

```diff
@@ -707,16 +708,20 @@ func TestServer_MapResultsToAtts(t *testing.T) {
 
 	resultsToAtts, err := ds.mapResultsToAtts(ctx, results)
 	require.NoError(t, err)
-	if !reflect.DeepEqual(expectedResultsToAtts, resultsToAtts) {
-		t.Error("Expected map:")
-		for key, value := range resultsToAtts {
-			t.Errorf("Key %#x: %d atts", key, len(value))
-			t.Errorf("%+v", value)
-		}
-		t.Error("To equal:")
-		for key, value := range expectedResultsToAtts {
-			t.Errorf("Key %#x: %d atts", key, len(value))
-			t.Errorf("%+v", value)
+	for k := range expectedResultsToAtts {
```

Comment:
Changed this to for loop the keys because:
- Since changing to protobuf v2, reflect.DeepEqual has problems due to the unexported fields.
- sszutil.DeepEqual doesn't support maps

This essentially means we have no simple way of checking equality on maps. Is this a problem?